test/album.js

```diff
@@ -370,4 +371,102 @@ describe('albums', () => {
       });
     });
   });
+  describe('/users/albums/:id/photos GET', () => {
+    it('should list all albums photos without problems because user purchased the album', done => {
+      const [mockedPhoto] = photos,
+        [mockedAlbum] = albums;
+
+      nock(`${url}/1`)
+        .get('')
+        .reply(200, mockedAlbum);
+
+      createUser(userOne).then(() => {
+        login({
+          email: 'sarahi12hg@wolox.com',
+          password: 'woloxwoloA1520'
+        }).then(res => {
+          chai
+            .request(server)
+            .post('/albums/1')
+            .set(config.common.session.header_name, res.headers[config.common.session.header_name])
+            .send()
+            .then(() => {
+              nock(`${url}/1/photos`)
+                .get('')
+                .reply(200, mockedPhoto);
+
+              chai
+                .request(server)
+                .get(`/users/albums/1/photos`)
+                .set(config.common.session.header_name, res.headers[config.common.session.header_name])
+                .then(async result => {
+                  expect(result).have.status(200);
+                  expect(result.body.id).to.be.equal(1);
+                  dictum.chai(result, 'list all album photos from purchased albums by user');
+                  done();
+                });
+            });
+        });
+      });
+    });
+
+    it('should fail list album photos because the user do not have this album yet', done => {
```

Comment:
Add a test with some albums of other user and check this, admin user case is missing too